zapisy/apps/common/templatetags/filters.py

```diff
@@ -36,7 +38,12 @@ def minimum(first, second):
     return min(first, second)
 
 
-@register.simple_tag(name='markdown', takes_context=True)
+@register.filter
+def to_json(dictionary: dict) -> str:
+    return json.dumps(dictionary)
+
+
+@register.simple_tag(name="markdown", takes_context=True)
```

Comment:
Już o tym gdzieś pisałem…
```suggestion
@register.simple_tag(name='markdown', takes_context=True)
```